Tests/js/views/services/assets/ez-locationviewviewservice-tests.js

```diff
@@ -478,9 +479,316 @@ YUI.add('ez-locationviewviewservice-tests', function (Y) {
+                        "The notification text should contain content name"
+                    );
+                    Assert.isTrue(
+                        e.notification.identifier.indexOf(locationId) !== -1,
+                        "The notification identifier should contain location id"
+                    );
+                    Assert.areSame(
+                        5, e.notification.timeout,
+                        "The notification timeout should be set to 0"
+                    );
+                });
+            });
+
+            this.service.fire('whatever:sendToTrashAction', {content: content});
+            Assert.isTrue(notified, "The notified event should have been fired");
+        },
+
+        "Should navigate the app to view of parent location": function () {
+            var content = new Mock(),
+                parentLocation = new Mock(),
+                locationId = 'raul-gonzalez-blanco',
+                contentName = 'pierlugi-collina',
+                parentLocationId = 'alan-shearer';
+
+            this.service.set('path', [{location: parentLocation}]);
+
+            Mock.expect(this.location, {
+                method: 'get',
+                args: ['id'],
+                returns: locationId,
+            });
+
+            Mock.expect(content, {
+                method: 'get',
+                args: ['name'],
+                returns: contentName,
+            });
+
+            Mock.expect(parentLocation, {
+                method: 'get',
+                args: ['id'],
+                returns: parentLocationId
+            });
+
+            Mock.expect(this.location, {
+                method: 'trash',
+                args: [Mock.Value.Object, Mock.Value.Function],
+                run: function (options, callback){
+                    callback(false);
+                }
+            });
+
+            Mock.expect(this.app, {
+                method: 'navigateTo',
+                args: [Mock.Value.String, Mock.Value.Object],
+                run: function (routeName, params) {
+                    Assert.areEqual(
+                        routeName, 'viewLocation',
+                        "Route name passed to navigateTo should be set to 'viewLocation'"
+                    );
+                    Assert.isObject(params, "Params passed to navigateTo should be an object");
+                    Assert.isString(params.id, "Params should provide id of location");
+                    Assert.areEqual(
+                        params.id, parentLocationId,
+                        "Id of location passed to navigateTo method should be the parent location id"
+                    );
+                }
+            });
+
+            this.service.on('confirmBoxOpen', function (e) {
+                e.config.confirmHandler.apply(this);
+            });
+
+            this.service.fire('whatever:sendToTrashAction', {content: content});
```

Comment:
you should also add a `Mock.verify(this.app)` here to check that we really redirect the user.